### ISSUE TITLE
nuclear vs mitochondrial subtypes patterns

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1904]
- match current nuclear subtype or mito subtype pattern, results in TSV (DOSDP query) 
- delete existing logical definitions
- generate from TSV the new logical definitions with the correct pattern (new pattern)

**Example:** 
MONDO_0007415 'mitochondrial complex III deficiency nuclear type 1'
```
'mitochondrial complex III deficiency'
 and ('disease has basis in dysfunction of' some BCS1L)
```

To new pattern:
```
'mitochondrial complex III deficiency'
 and ('disease has basis in dysfunction of' some (BCS1L and ('part of' some nucleus)))
```

@cmungall

**Comment by cmungall** [comment]
I don't think we should use nesting, and we don't need it here. The organellar origin is a property of the gene already

**Comment by sabrinatoro** [comment]
This issue was created more than 2 years ago and has not been addressed to date. It has therefore been identified as a candidate to be closed without action.
If anyone believes that this issue should be addressed, please comment here.
Otherwise, this issue will automatically be closed with the October 2025 release.

Note: @matentzn @nicolevasilevsky  IMO, in order to keep this issue open, we need to have an explanation of the reasons for doing this. Without a use case, I don't think it is worth our time to do it.
